stdlib/builtins.pyi

```diff
@@ -98,7 +98,6 @@ class object:
     __annotations__: dict[str, Any]
     @property
     def __class__(self) -> type[Self]: ...
-    # Ignore errors about type mismatch between property getter and setter
```

Comment:
Noticed by @Akuli in https://github.com/python/typeshed/pull/11496#discussion_r1508111612
Added in #5092